diff -U 3 -H -d -r -N -- a/kadu-core/CMakeLists.txt b/kadu-core/CMakeLists.txt
--- a/kadu-core/CMakeLists.txt	2012-09-14 23:54:35.000000000 +0200
+++ b/kadu-core/CMakeLists.txt	2020-11-01 21:19:13.075003972 +0100
@@ -158,7 +158,7 @@
 include_directories (${QCA2_INCLUDE_DIRS})
 
 if (UNIX AND NOT APPLE)
-	find_package (X11 REQUIRED COMPONENTS X11 Xutil Xfixes)
+	find_package (X11 REQUIRED COMPONENTS Xutil Xfixes)
 	if (X11_FOUND AND X11_Xutil_FOUND AND X11_Xfixes_FOUND)
 		message (STATUS "Found required X11 libraries: ${X11_X11_LIB};${X11_Xfixes_LIB}")
 	else (X11_FOUND AND X11_Xutil_FOUND AND X11_Xfixes_FOUND)
diff -U 3 -H -d -r -N -- a/plugins/idle/CMakeLists.txt b/plugins/idle/CMakeLists.txt
--- a/plugins/idle/CMakeLists.txt	2012-09-14 23:54:35.000000000 +0200
+++ b/plugins/idle/CMakeLists.txt	2020-11-01 21:20:55.068434413 +0100
@@ -14,7 +14,7 @@
 )
 
 if (UNIX AND NOT APPLE)
-	find_package (X11 REQUIRED COMPONENTS X11 Xutil Xscreensaver)
+	find_package (X11 REQUIRED COMPONENTS Xutil Xscreensaver)
 	if (X11_Xscreensaver_FOUND)
 		message (STATUS "Found required X11 libraries: ${X11_X11_LIB};${X11_Xscreensaver_LIB}")
 	else (X11_Xscreensaver_FOUND)
diff -U 3 -H -d -r -N -- a/plugins/pcspeaker/CMakeLists.txt b/plugins/pcspeaker/CMakeLists.txt
--- a/plugins/pcspeaker/CMakeLists.txt	2012-09-14 23:54:36.000000000 +0200
+++ b/plugins/pcspeaker/CMakeLists.txt	2020-11-01 21:21:47.551692689 +0100
@@ -25,7 +25,7 @@
 set (LIBRARIES "")
 
 if (UNIX AND NOT APPLE)
-	find_package (X11 REQUIRED COMPONENTS X11)
+	find_package (X11 REQUIRED)
 	if (NOT X11_FOUND)
 		message (SEND_ERROR "Could NOT find X11")
 	endif (NOT X11_FOUND)
diff -U 3 -H -d -r -N -- a/plugins/screenshot/CMakeLists.txt b/plugins/screenshot/CMakeLists.txt
--- a/plugins/screenshot/CMakeLists.txt	2012-09-14 23:54:36.000000000 +0200
+++ b/plugins/screenshot/CMakeLists.txt	2020-11-01 21:22:22.175532516 +0100
@@ -46,7 +46,7 @@
 )
 
 if (UNIX AND NOT APPLE)
-	find_package (X11 REQUIRED COMPONENTS X11 Xext Xshape)
+	find_package (X11 REQUIRED COMPONENTS Xext Xshape)
 	if (X11_Xshape_FOUND)
 		message (STATUS "Found required X11 libraries: ${X11_X11_LIB};${X11_Xext_LIB}")
 	else (X11_Xshape_FOUND)
